src/main/java/com/tileman/TilemanModeConfig.java

```diff
@@ -94,14 +97,37 @@ default int warningLimit() {
         return 50;
     }
 
+    @ConfigItem(
+            keyName = "enableTilesWarning",
+            name = "Enable Tiles Warning",
+            section = settingsSection,
+            description = "Turns on tile warnings when you reach your set limit or 0.",
+            position = 4
+    )
+    default boolean enableTileWarnings() {
+        return true;
```

Comment:
Lets default this to false and then make the number default 20

Reply:
Done